dynamic-programming/fibonacci_number.cpp

```diff
@@ -1,18 +1,30 @@
 #include <iostream>
+
 int fibonacci_fast(int n) {
     // write your code here
-    int fin[n];
+    if ( n == 0 ) {
+        return 0;
+    }
+    int fin[3];
```

Comment:
Instead of the array you can write simple variables. But this isn't a dynamic algorithm. You can send your algorithm as a own pull request. Other approach.